framework/include/mediaplayer/mediaplayer.h

```diff
@@ -0,0 +1,224 @@
+#ifndef __MEDIAPLAYER_H__
+#define __MEDIAPLAYER_H__
+
+#include <queue.h>
+#include <sys/types.h>
+
+
+
+/***
+ * Enumerations
+ */
+
+typedef enum {
```

Comment:
[R11] Don’t declare both a structure tag and variables or typedefs in the same declaration.

[Example-Do]
```
struct example{
    int lock;
    const char *device_id;
};
static struct example example_busy[CHANNELS] = {
    [4] = { 1, "cascade" },
};
```

[Example-Don't]
```
struct example{
    int lock;
    const char *device_id;
}example_busy[CHANNELS] = {
    [4] = { 1, "cascade" },
};

typedef struct example_t{
    int lock;
    const char *device_id;
}example;
```

Reply:
done